code/game/objects/items/weapons/grenades/frag.dm

```diff
@@ -37,24 +39,24 @@
 
 	var/mob/living/carbon/human/H = target
 	if(!prob(embed_prob - ARMOUR_VALUE_TO_PERCENTAGE(H.getarmor(null, BOMB))))
-		to_chat(H, "<span class='warning'>Shrapnel bounces off your armor!</span>")
+		to_chat(H, "<span class='warning'>An explosive fragment bounces off your armor!</span>")
```

Comment:
Is the fragment explosive? This doesn't seem like a proper sentence to me

Reply:
Altered.